modules/zivid/captureassistant.py

```diff
@@ -0,0 +1,82 @@
+"""Contains capture assistant functions and classes."""
+import _zivid
+import zivid._settings_converter as _settings_converter
+from zivid._make_enum_wrapper import _make_enum_wrapper
+
+
+AmbientLightFrequency = _make_enum_wrapper(
+    _zivid.captureassistant.AmbientLightFrequency,
+    "Ensure compatibility with the frequency of the ambient light in the scene.",
+)
```

Comment:
Very nice, very clean :)
Auto-completion of options also seems to work everywhere.

Reply:
Nice :man_dancing: